src/main/java/antlr/treepat/TreepatVisitorImplementation.kt

```diff
@@ -49,7 +50,7 @@ class TreepatVisitorImplementation : TreepatVisitor<ASTNode> {
     }
 
     override fun visitIndent(ctx: TreepatParser.IndentContext): ASTNode {
-        return Check(ctx.treepat().accept(this))
+        return Check(ctx.subtree().accept(this))
```

Comment:
Lets simplify the functions that just have one return by doing:
```
override fun visitIndent(ctx: TreepatParser.IndentContext): ASTNode = Check(ctx.subtree().accept(this))
```
Same for the ones bellow.

Reply:
Ok, I also orginized the file so all one line functions are together.